app/models/company.rb

```diff
@@ -1,6 +1,8 @@
 class Company < ActiveRecord::Base
   include ActiveModel::Dirty
 
+  has_many :notes
```

Comment:
This wasn't fully specified, so it's interesting to see how you implemented notes. I was envisioning that each company had one note associated. Let's talk about implementation at the stand up tomorrow. 
Here are links to other products with the two options. I think has_one is more common, as far as I could see from a quick google search.
has_many notes: http://www.actpresentation.com/shots/quick_nav_list1.jpg
has_one notes: http://databasemelbourne.com/wp-content/uploads/2013/09/contactdetails.png